app/core/Router.coffee

```diff
@@ -72,15 +72,15 @@ module.exports = class CocoRouter extends Backbone.Router
     'contribute/diplomat': go('contribute/DiplomatView')
     'contribute/scribe': go('contribute/ScribeView')
 
-    'courses': go('courses/CoursesView')
-    'Courses': go('courses/CoursesView')
-    'courses/students': redirect('/courses')
+    'courses': redirect('/students') # Redirected 9/3/16
+    'Courses': redirect('/students') # Redirected 9/3/16
```

Comment:
Also it's also weird that if you don't have the query param included, CoursesView (`/students`) shows the error screen. Shouldn't `/students` look the same if you have no role set, regardless of GET query parameters?

https://github.com/codecombat/codecombat/blob/master/app/views/courses/CoursesView.coffee#L70-L71